packages/manager/modules/netapp/src/dashboard/template.html

```diff
@@ -59,7 +59,7 @@
                     </oui-tile-definition>
                     <oui-tile-definition
                         term="{{:: 'netapp_dashboard_information_quota' | translate }}"
-                        description="{{ 'netapp_dashboard_information_quota_formatted' | translate:{quota: $ctrl.storage.quota } }}"
+                        description="{{ $ctrl.storage.quota | cucBytes:0:false:'GB' }}"
```

Comment:
**issue(blocking):** `cucBytes` filter come from `@ovh-ux/ng-ovh-cloud-universe-components`.
IMHO it doesn't make sense to have it as a peerDependency since NetApp is associated with Bare Metal Cloud section.

Reply:
I am not sure to ge your point of view.
Currently the `"@ovh-ux/ng-ovh-cloud-universe-components"` is only declared as dependency into `packages/manager/apps/dedicated/package.json` dependecies node.
So regarding others modules (manager/modules/*), some modules which required the `cucBytes` add it as peerDependency.

What is the issue and do you have a suggestions ?

Reply:
Did you start the standalone application `@ovh-ux/manager-netapp-app` (which is highly recommend)?
You'll noticed that `cucBytes` filter isn't working.

Reply:
@mohammed-zahaf it might be interesting to look at the filter located here: https://github.com/ovh/manager/tree/master/packages/manager/modules/filters/src/bandwidth. This way you could avoid relying on `@ovh-ux/ng-ovh-cloud-universe-component`

Reply:
Thanks i add the filter module as deps.

Reply:
**question:** Couldn't be replaced by filter suggested by @JeremyDec?

Reply:
Ah i was think this module have a `cucBytes` filter.
So i am note sure this module give the right unit value.

This filter is for bandwidth (B/s) and i have to convert a quota (B).
There is a way to achieve my task with the bandwidth filter ?

Reply:
As we discuss together @JeremyDec, i changed the the `cucBytes` by `bytes`.